tests/src/test/spark330/scala/org/apache/spark/sql/rapids/utils/RapidsSQLTestsBaseTrait.scala

```diff
@@ -121,6 +121,12 @@ object RapidsSQLTestsBaseTrait {
         "org.apache.spark.sql.rapids.ExecutionPlanCaptureCallback")
       .set("spark.sql.warehouse.dir", warehouse)
       .set("spark.sql.cache.serializer", "com.nvidia.spark.ParquetCachedBatchSerializer")
+      .set("spark.sql.session.timeZone", "UTC")
+      .set("spark.rapids.sql.explain", "ALL")
```

Comment:
This will make the tests very verbose.  Intentional, or debug code accidentally left in?

Reply:
both are Intentional.

- For "spark.sql.session.timeZone", we find many expressions won't go to GPU routine unless timezone is set to UTC. Since we want to test as many test cases as possible in GPU, we'd better use UTC timezone

- For "spark.rapids.sql.explain", since by default tests/src/test/resources/log4j2.properties already set console appender's log level to error, we won't see verbose logs even if spark.rapids.sql.explain is set to ALL here. The benifit of setting it to ALL is, when we want to check&debug test cases, we only need to change log4j2.proerties, instead of changing both log4j2.proerties and adding `.set("spark.rapids.sql.explain", "ALL")` in code

Reply:
> Since we want to test as many test cases as possible in GPU, we'd better use UTC timezone

I'm not questioning that we need to control the timezone for these tests, but rather the discussion is about _how_ to control the timezone.  Normally we don't want hardcoding like this, because now the test _cannot_ check if we're doing the right thing in any timezone other that UTC, and like I said, we do have some timezone support with more on the way.  Hardcoding this precludes using these tests to verify we're doing the right thing in any other timezone.  Having the test environment setup script control that makes the test reusable.